### PR TITLE
AO3-5371 Add descriptive classes to external work and series blurbs

### PR DESCRIPTION
**Pull request by weeklies**
# Pull Request Checklist

* [x] Have you read ["How to write the perfect pull request"](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)?
* [x] Have you read the [contributing guidelines](https://github.com/otwcode/otwarchive/blob/master/CONTRIBUTING.md)?
* [x] Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
* [x] Have you added the [Jira](https://otwarchive.atlassian.net) issue number
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
* [x] Do you have fewer than 5 pull requests already open? If not, please wait
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-5371

## Purpose

Add descriptive classes to series and external work blurbs. This will be reflected across different pages, including for bookmarks.

I also tried to do some internationalization in the lines I changed, and added the number delimiter to `link_to_bookmarkable_bookmarks` in `bookmarks_helper.rb`. I hope it's not out of scope?

Some points of uncertainty: 
1. I added an `inspired` class for the related works statistic — not sure about this and can revert this back if necessary.
 2. If a descriptive class is needed for the `complete` status  in series, please let me know.

## Testing Instructions

Try to create a site skin that changes the color of soecfic statistics fields in series and external work blurbs. For example:

```
.stats dt.bookmarks a,
.stats dt.words,
.stats dt.works,
.stats dt.inspired,
.stats dd.bookmarks a,
.stats dd.words,
.stats dd.works,
.stats dd.inspired {
  font-weight: bold;
  color: red;
}
```

 Make sure it works as intended.

## Credit
weeklies (she/her)

### CODE REVIEW

**Review by sarken — COMMENTED** [review]
The delimiter change is fine! We should just add testing info for it.

I had one quick question and just a general note for the future.

**Comment by sarken** [comment]
...oh, how interesting, I clicked Cancel and it posted. 🤬  I might have one more comment, if you'd like to wait -- I'll let you know.

File: config/locales/views/en.yml [hunk 1]
```diff
@@ -525,6 +525,11 @@ en:
   skins:
     confirm_delete:
       confirm_html: Are you sure you want to <strong><em>delete</em></strong> the skin "%{skin_title}"?
+  stats:
```
Comment (sarken) [inline config/locales/views/en.yml:528]: Okay, yeah, here's the "one more" comment missing from my premature review:

Should these be attributes in the models locale file instead? I'm kind of shaky on my understanding of that, so I'll need someone else to weigh in, but I wanted to bring it up.

File: app/views/external_works/_blurb.html.erb [hunk 2]
```diff
@@ -42,12 +42,12 @@
...
     <% if Bookmark.count_visible_bookmarks(external_work) > 0 %>
-    <dt><%= ts("Bookmarks") + ": " %></dt>
-    <dd><%= link_to_bookmarkable_bookmarks(external_work) %></dd>
+    <dt class="bookmarks"><%= t("stats.bookmarks") + ":" %></dt>
+    <dd class="bookmarks"><%= link_to_bookmarkable_bookmarks(external_work) %></dd>
     <% end %>
 
-    <dt><%= ts("Related Works") + ": "%></dt>
-    <dd><%= link_to external_work.related_works.count.to_s, external_work  %></dd>
+    <dt class="inspired"><%= ts("stats.related_works") + ":" %></dt>
```
[the remark this answers is not in the document]
Comment (weeklies) [reply app/views/external_works/_blurb.html.erb:49]: I wasn't sure whether the class should be `related_works` or whether `related` sounded too vague —but I can change it, if it is clearer?

[the remark this answers is not in the document]
Comment (sarken) [reply app/views/external_works/_blurb.html.erb:49]: Nah, inspired is fine!

**Comment by weeklies** [comment]
Temporarily closing this because I would like to submit a PR for a higher priority issue.

File: config/locales/views/en.yml (hunk 1 above)
Comment (weeklies) [reply config/locales/views/en.yml]: I've tried to shift it to the models now.

**Comment by weeklies** [comment]
Closing this (again) this because I would like to submit a PR that updates a stale issue.

**Review by brianjaustin — APPROVED** [review]

**Pull request merged**